tools/diff-processor/cmd/detect_missing_docs_test.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/magic-modules/tools/diff-processor/detector"
+	"github.com/GoogleCloudPlatform/magic-modules/tools/diff-processor/diff"
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func TestDetectMissingDocs(t *testing.T) {
```

Comment:
Can we add some more complex cases? Like nested objects?

Reply:
I added the tests in detector_test.go, and leaving this to mostly verify the output is the expected json format.